<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmendPWDialog</class>
 <widget class="QDialog" name="AmendPWDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>密码修改</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="userLabel">
           <property name="text">
            <string>用户名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="passworLable">
           <property name="text">
            <string>密  码：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>新密码：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>确认新密码：</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="userLineEdit">
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pwLineEdit">
           <property name="maxLength">
            <number>16</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="newPWlineEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="newPwlineEdit2">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="amendOKButton">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="amendCancleButton">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
